lib/spack/docs/analyze.rst

```diff
@@ -59,7 +59,7 @@ are available:
     install_files            : install file listing read from install_manifest.json
     environment_variables    : environment variables parsed from spack-build-env.txt
     config_args              : config args loaded from spack-configure-args.txt
-    abigail                  : Application Binary Interface (ABI) features for objects
+    libabigail                  : Application Binary Interface (ABI) features for objects
```

Comment:
@vsoch This seems fine to me other than the OCD triggered by the misaligned `:` above :point_up: :laughing:

Reply:
I think it will render ok? I’ll check after and open another PR if it looks wonky.

Reply:
Ahah, yeah it probably renders ok - the OCD was for me seeing this in the `rst` 😆

Reply:
Oh gotcha! Well I'll follow up with a quick fix for that.